docs/_resources/contributors.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: default
+parent: Resources
+title: How to Contribute
+nav_order: 10
+description: 'Contribute with the FinOps toolkit'
+permalink: /resources/contributors
+---
+
+<span class="fs-9 d-block mb-4">FinOps Toolkit Contributors Guideline</span>
+The FinOps Toolkit project welcomes contributions from the community.
+{: .fs-6 .fw-300 }
+
+<details open markdown="1">
+   <summary class="fs-2 text-uppercase">On this page</summary>
+
+
+- [📝 How to Contribute](#-finops-toolkit-contribution-guide)
+  - [Working on existing bugs](#1-working-on-existing-bugs)
+  - [Suggesting and owning a new feature, fixing a bug, or suggesting a change for an existing Tool](#2-suggesting-and-owning-a-new-feature-fixing-a-bug-or-suggesting-a-change-for-an-existing-tool)
+  - [Submitting a proposal for a new tool](#3-submitting-a-proposal-for-a-new-tool)
+  - [Code review](#4-code-review)
+  - [Answering questions](#5-answering-questions)
+  - [Documentation update](#6-documentation-update)
+
+</details>
+
+---
+
+## 📝 FinOps Toolkit Contribution Guide
+
+Welcome to the FinOps Toolkit project! As an open-source initiative, our success hinges on the collective effort of the community. Your contributions play a vital role in enhancing and expanding the toolkit's capabilities, making it more valuable and effective.
+
+There are three primary ways you can contribute to the success of the project:
+
+1. 🐛 **Working on existing bugs:** You can assist by identifying and addressing existing bugs listed in our repository. 
+2. 🛠️ **Suggesting and owning a new feature, fixing an existing bug, or suggesting a change for an existing tool:** This involves proposing new features, fixing bugs, or suggesting changes for existing tools. 

```

Comment:
This is covering creating issues and making code changes. Should we split them for people who are may not be able to contribute code?